merge_sort/merge_sort.py

```diff
@@ -0,0 +1,65 @@
+def merge(array, left, right):
+    """
+    Perform Merge Operation between arrays.
+
+    :param array: Iterable of elements
+    :param left: left limit for merge sort
+    :param right: right limit for merge sort
+    :return: no returns, merges arrays.
+    """
+    mid = int((left + right )/ 2)
+    s1 = int(mid - left + 1)
+    s2 = int(right - mid)
+    l = [0] * s1
+    r = [0] * s2
+    for i in range(s1):
+        l[i] = array[int(left + i)]
+
+    for j in range(s2):
+        r[j] = array[mid + 1 + j]
+
+    i = j = int(0)
+    k = int(left)
+    while(i < s1 and j < s2):
+        if(l[i] <= r[j]):
+            array[k] = l[i]
+            i = i + 1
+        else:
+            array[k] = r[j]
+            j = j + 1
+
+        k = k + 1
+
+    while(i < s1):
+        array[k] = l[i]
+        k = k + 1
+        i = i + 1
+
+    while(j < s2):
+        array[k] = r[j]
+        k = k + 1
+        j = j + 1
+
+
+def sort(array, left, right):
+    """
```

Comment:
Driver function name is `merge_sort in previously added (Merge sort)implementations in c and java. Driver function should be merge_sort.
The implementation is good btw.
:)